### PR TITLE
I18N: Remove line breaks in sentence

### LINKED ISSUE #1070: I18n: Line breaks in multiple sentences string not rendered
This string is formated with line breaks that aren't used in the UI:
```
Here you can mass-create translation sets in this project.
The list of translation sets will be mirrored with the sets of a project you choose.
Usually this is one of the parent projects.
```
Source:
https://github.com/GlotPress/GlotPress-WP/blob/develop/gp-templates/project-mass-create-sets.php#L18-L20

Here is how it's rendered:
imagem

I suggest to separate in multiple strings, adding `<br>` in between for line breaks.

### PR DESCRIPTION
**Pull request by pedro-mendonca**
Fix #1070

### CODE REVIEW

File: gp-templates/project-mass-create-sets.php [hunk 1]
```diff
@@ -15,12 +15,14 @@
 gp_tmpl_header();
 ?>
 <h2><?php _e( 'Mass-create Translation Sets', 'glotpress' ); ?></h2>
-<p><?php _e( 'Here you can mass-create translation sets in this project.
-The list of translation sets will be mirrored with the sets of a project you choose.
-Usually this is one of the parent projects.', 'glotpress' ); ?></p>
+<p>
+	<?php _e( 'Here you can mass-create translation sets in this project.', 'glotpress' ); ?><br>
+	<?php _e( 'The list of translation sets will be mirrored with the sets of a project you choose.', 'glotpress' ); ?><br>
+	<?php _e( 'Usually this is one of the parent projects.', 'glotpress' ); ?>
```
Comment (ocean90) [inline gp-templates/project-mass-create-sets.php]: I think these two lines need to stay together without a line break so you know what "this" refers to.

Comment (pedro-mendonca) [reply gp-templates/project-mass-create-sets.php]: So, merging both with a `<br>` tag in the string?

Comment (ocean90) [reply gp-templates/project-mass-create-sets.php]: No, without any line break.

Comment (pedro-mendonca) [reply gp-templates/project-mass-create-sets.php]: I agree with that. Pushing it in a minute.

**Review by ocean90 — APPROVED** [review]

**Pull request merged**